climada/test/test_api_client.py

```diff
@@ -35,38 +35,40 @@ class TestClient(unittest.TestCase):
 
     def test_data_type(self):
         """"""
-        lpdt = Client().get_data_type("litpop")
+        lpdt = Client().get_data_type_info("litpop")
         self.assertEqual(lpdt.data_type, 'litpop')
         self.assertEqual(lpdt.data_type_group, 'exposures')
+        self.assertTrue('climada_version' in [p['property'] for p in lpdt.properties if p['mandatory']])
+        self.assertTrue('country_name' in [p['property'] for p in lpdt.properties if not p['mandatory']])
 
     def test_data_types(self):
         """"""
-        exdts = Client().get_data_types("exposures")
+        exdts = Client().list_data_type_infos("exposures")
         self.assertTrue('litpop' in [exdt.data_type for exdt in exdts])
 
     def test_datasets(self):
         """"""
-        datasets = Client().get_datasets(status=None, name='FAOSTAT_data_producer_prices')
+        datasets = Client().list_dataset_infos(status=None, name='FAOSTAT_data_producer_prices')
         self.assertEqual(len(datasets), 1)
 
     def test_dataset(self):
         """"""
         client = Client()
 
-        dataset = client.get_dataset(name='FAOSTAT_data_producer_prices')
+        dataset = client.get_dataset_info(name='FAOSTAT_data_producer_prices', status='test_dataset')
         self.assertEqual(dataset.version, 'v1')
         self.assertEqual(len(dataset.files), 1)
         self.assertEqual(dataset.files[0].file_size, 26481)
-        self.assertEqual(dataset.data_type.data_type, 'crop_production')
+        self.assertEqual(dataset.data_type, DataTypeShortInfo('crop_production', 'exposures'))
```

Comment:
What is `DataTypeShortInfo` ?

Reply:
A @dataclass containing data_type and data_type_group, without description, status and property list.